### ISSUE TITLE
Empty default value in cli options

### ISSUE THREAD

**Issue by outsideris** [mochajs/mocha#3433]
```
--globals <names>                       allow the given comma-delimited global [names] (default: )
```

If default value is an empty list(`[]`), it shows as `(default: )`. It is confusing to users. No default or `(default: [])` is better.

**Comment by outsideris** [comment]
If [commander fix it](https://github.com/tj/commander.js/pull/856) and then a new version released, we can fix this.

**Comment by plroebuck** [comment]
Hey, they finally released the fix in v2.19.0.
